06 - Type Ahead/index-START.html

```diff
@@ -15,8 +15,47 @@
     </ul>
   </form>
 <script>
-const endpoint = 'https://gist.githubusercontent.com/Miserlou/c5cd8364bf9b2420bb29/raw/2bf258763cdddd704f8ffd3ea9a3e81d25e2c6f6/cities.json';
+  const endpoint = 'https://gist.githubusercontent.com/Miserlou/c5cd8364bf9b2420bb29/raw/2bf258763cdddd704f8ffd3ea9a3e81d25e2c6f6/cities.json';
 
+  const cities = [];
+
+  fetch(endpoint)
+    .then(e => e.json())
+    .then(data => cities.push(...data))
```

Comment:
This works, but depending on the internet's speed, the user may be able to interact with the UI before the fetch has been resolved. You can either put all your JS inside the second then, or use `async` and `await`